domains/foodfinderga.org/html/css/front/bootstrap.css

```diff
@@ -64,8 +64,11 @@
 .footer-text { color:#FFF; margin-top: 2%;padding-bottom:1%; font-size:16px}
 #fixed-div { position:absolute;top:0;right:0; }
 .searchschool_form{
+    display:block;
```

Comment:
good ole display block.  that was one of the things I was trying to remember when we were working on it the other day